operators/specifying-a-requirements.txt.md

```diff
@@ -1,10 +1,6 @@
 # Specifying a requirements.txt
 
-When a function is annotated with `@op`, Aqueduct automatically attempts to infer the Python library requirements for that function by looking at your development environment. We do this by using a tool called [pipreqsnb](https://pypi.org/project/pipreqsnb/).
-
-For a variety of reasons, this might not always be sufficient. If you'd like to specify your own custom `requirements.txt`, Aqueduct will prioritize those requirements over the default inferred requirements.&#x20;
-
-The simplest way to specify a `requirements.txt` is to create one in the directory where you're running the Aqueduct SDK. When Aqueduct detects that a `requirements.txt` is present, it will automatically use this file instead of the inferred requirements.
+The simplest way to specify a `requirements.txt` is to create one in the directory where you're running the Aqueduct SDK. When Aqueduct detects that a `requirements.txt` is present, it will automatically use this file to determine the requirements.
```

Comment:
@Fanjia-Yan has a PR out to specify function-specific requirements.txt. Can we update the documentation to reflect that as well?

Reply:
This PR is almost out, just wrapped up the integration tests for the functionalities today and will ask Kenny for some reviews and suggestions, and then it's all set to go.

Reply:
Should be able to merge it tomorrow hopefully